Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,51 @@
+
+#### Playbooks
+
+##### New: Suspicious certutil command line
+
+This playbook handles "Suspicious certutil command line" alerts.
+
+Playbook Stages:
+
+Analysis:
+
+During the alert analysis, the playbook will perform the following actions:
+
+- Enrich and checks if the URL reputation is suspicious.
+- Checks if any process in the causality chain is unsigned.
+- Checks if any process in the causality chain is non-prevalent.
+- Searches for Cortex XSIAM agent alerts related to file drops using certutil.
+- Checks for any suspicious parameters in the command line (if the command line risk score is medium or higher).
+
+If the playbook detects any of these conditions, it will proceed to the early containment stage; otherwise, it will close the alert.
+
+Early Containment:
+
+- Identify if an agent prevention rule was triggered. If triggered in block mode, proceed with the URL reputation check; otherwise, terminate the causality process tree.
+
+Verdict:
+
+- Based on the URL's reputation, if it is found to be malicious, the playbook will perform remediation actions; otherwise, it will close the alert.
```

Comment:
Based on the URL's reputation, if found to be malicious, the playbook will perform remediation actions; otherwise, it will close the alert.
Comment